src/ChipCarousel/index.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { ForwardedRef } from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import classNames from 'classnames';
+// @ts-ignore
+import { OverflowScroll, OverflowScrollContext } from '../OverflowScroll';
```

Comment:
Let's try to ignore the necessary import rule for the whole file, this will help to avoid numerous `@ts-ignore`

Reply:
I would do it as a separate PR

Reply:
[clarification] Once the Paragon types PR merges in, I'm assuming these `@ts-ignore` comment would become unnecessary?